crates/swc/Cargo.toml

```diff
@@ -91,6 +91,7 @@ rayon = "1"
 swc_node_base = {version = "0.5.0", path = "../swc_node_base"}
 testing = {version = "0.18.0", path = "../testing"}
 walkdir = "2"
+swc_ecma_lints = {path = "../swc_ecma_lints", features = ["non_critical_lints"]}
```

Comment:
Please add version here, otherwise `cargo` will deny publishing.

Reply:
oops. missed it